### ISSUE TITLE
Update list of institutions with signed data agreements

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1162]
# TL;DR
The list of institutions with agreements is quite out of date! We need to update it prior to sending the CHS announcements!

Here is the new text for this section of the "The Scientists" static page - the only structural difference is that I've split the list by country (US, Canada, UK, "Worldwide", "Coming Soon").  Each group should be its own section (3 columns of bullets in wide view). The country labels should be not-very-prominent; they are just there to help people find their own institution more quickly

---

As of March 2023, there are over 950 researchers in our community, and over 90 universities affiliated with Lookit, including:

United States:

American University
Boston Children’s Hospital
Boston College
Boston University
Brown University
Bucknell University
Clark University
Cornell University
Davidson College
Duke University
Duke University School of Medicine
East Tennessee State University
Emory University
Florida International University
Fordham University
Franklin & Marshall College
George Mason University
Harvard University
Harvard Graduate School of Education
Haskins Laboratories, Yale University
Hope College
Kennedy Kreiger Institute, Johns Hopkins University
Kent State University
Loyola University Chicago
Massachusetts Institute of Technology
Montclair State University
Morgan State University
New York University
Northeastern University
Northern Illinois University
Northwestern University
Rutgers University - New Brunswick
Rutgers University - Newark
Sacred Heart University
Skidmore College
Smith College
Stanford University
Stony Brook University
The New School
University at Buffalo, The State University of New York
University of California, Berkeley
University of California, Davis
University of California, Los Angeles
University of California, Riverside
University of California, San Diego
University of Chicago
University of Illinois, Urbana-Champaign
University of Kansas
University of Louisville
University of Maryland, College Park
University of Massachusetts, Lowell
University of Massachusetts, Amherst
University of North Carolina, Asheville
University of Oregon
University of Pennsylvania
University of South Florida
University of Southern California
University of Texas, Austin
University of Texas, Dallas
University of Virginia
University of Washington
University of Wisconsin, Madison
Virginia Tech
Wellesley College
Williams College
Yale School of Medicine
Yale University
Yeshiva University - Stern College for Women

Canada:

Brock University
Concordia University
Dalhousie University
McGill University
McMasters University
Ryerson University
University of British Columbia
University of Toronto
University of Waterloo

United Kingdom:

Birkbeck, University of London
Goldsmiths, University of London
Great Ormond Street Institute of Child Health (UCL)
Nottingham University
St Andrews University
Ulster University
University College London
University of East Anglia
University of Edinburgh
University of Essex
University of Lincoln

Worldwide:

Aix-Marseille Université/CNRS (France)
Boğaziçi University (Turkey)
CEMIC-CONICET (Argentina)
Leiden University (Netherlands)
Max Planck Institute for Human Cognitive and Brain Sciences (Germany)
University of Tokyo 東京大学 (Japan)
Western Sydney University (Australia)

Coming soon (Agreements in process):

Carnegie Mellon University
Florida State University
Hunter College
Indiana University
John Jay College of Criminal Justice and CUNY Graduate Center
Max Planck Institute for Psycholinguistics
Mount Sinai Hospital
Oklahoma State University
Oxford University
Temple University
University of Arkansas
University of Cambridge
University of Minnesota, Twin Cities
University of North Carolina, Greensboro
University of Pittsburgh
Vanderbilt University
Villanova University
Western Washington University

To learn about joining us, visit the [guide to getting started!](https://lookit.readthedocs.io/en/develop/researchers-start-here.html)
